app/server/methods/notesMethods.js

```diff
@@ -0,0 +1,19 @@
+Meteor.methods({
+  updateApplicationNote: function(noteText, applicationId) {
+    check(noteText, String);
+    check(applicationId, String);
+
+    if (!this.userId) {
+      throw new Meteor.Error('error-not-signed-in', 'You must sign in first.', '654');
+    }
+
+    if (!Roles.userIsInRole(this.userId, ['committee',])) {
+      throw new Meteor.Error('not-allowed', 'You must be a committee user aka No Juice Error');
+    }
+
+    // Update note
+    Notes.update({userId: Meteor.userId(), applicationId: applicationId,}, { $set: { note: noteText } }, { upsert: true});
```

Comment:
This matches existing notes and replaces data even though no Note ID is supplied? And this creates new notes if not created?